docs/release-notes.rst

```diff
@@ -154,6 +154,14 @@ Functionality releated to displaying a sub-form based on some checkbox selection
 
 ------
 
+**'bigip-fast-templates/microsoft exchange' is not supported in F5 ACI ServiceCenter**
+
+If **bigip-fast-templates/microsoft exchange** template is used from the Basic sub-tab of **L4-L7 App Services --> Application**, you may receive a 'Null exception' from BIG-IP.
+
+**Workaround:** Create the microsoft exchange FAST application from BIG-IP's FAST UI.
```

Comment:
Create the Microsoft Exchange FAST application...

<also, it seems like the first two references to the template (bigip-fast-templates/microsoft exchange) are paths, and if they are, there shouldn't be a space between microsoft and exchange, maybe an underscore, etc.  If not a path, then Microsoft and Exchange should also be capitalized.>